influx_aggregator/aggregator.py

```diff
@@ -8,9 +8,9 @@
 import time
 
 # You can generate an API token from the "API Tokens Tab" in the UI
-token = "bNMiIcMJcrP5x5YQM0D91Rod2hakW_aodrFfr6yPbN4crDJbJw4dmb_cgwclH80vql6_nOhih7MHiwkLDztcfA=="
-org = "UCR Solar Car"
-bucket = "Telemetry"
+token = "uthxvFH-S6OLkkpzGWxqCjA9AydTZPRA2pgUVI_BFyQzA6cLKjGgQpa2tqitYItDw11wxptYeJK65fZ1xmFyfg=="
```

Comment:
read the values from the `.env.dev` file instead, read the token, org, and bucket